scipy-notebook/Dockerfile

```diff
@@ -74,6 +74,10 @@ RUN conda create --quiet --yes -p $CONDA_DIR/envs/python2 python=2.7 \
 RUN ln -s $CONDA_DIR/envs/python2/bin/pip $CONDA_DIR/bin/pip2 && \
     ln -s $CONDA_DIR/bin/pip $CONDA_DIR/bin/pip3
 
+# Import matplotlib the first time to build the font cache.
+ENV XDG_CACHE_HOME /home/$NB_USER/.cache/
+RUN $CONDA_DIR/envs/python2/bin/python -c "import matplotlib.pyplot"
```

Comment:
Travis is failing because it's trying to use the qt backend by default. If you move these lines below line 83 where the import hook makes matplotlib inline the default and use ipython to run the import instead of python, it might work.